content-src/experiments/lockbox.yaml

```diff
@@ -125,4 +125,4 @@ contributors:
 privacy_preamble: >
   Firefox Lockbox allows Firefox Accounts users to access the login and password information (which we call “credentials”) on their mobile phones, outside any browser, by using Firefox Sync. In addition to the data collected by all Test Pilot experiments...
```

Comment:
>>In addition to the data collected by all Test Pilot experiments...

>Shouldn't really be there

This was a creative solution to including required legal language while simultaneously removing it since we don't have "measurements" for this experiment:

![screen shot 2018-06-30 at 7 39 25 am](https://user-images.githubusercontent.com/49511/42125692-bc401d10-7c38-11e8-9394-3e32cb93cd6f.png)

So while its redundant, I don't have a good way to include that sentence other than adding it to this preamble. 😔

Reply:
> So while its redundant, I don't have a good way to include that sentence other than adding it to this preamble.

Why do you need a preamble at all? The link is visually separated.

Reply:
Yeah, Legal has asked to include a) that blurb that starts with "Firefox Lockbox allows..." and also b) the note about the other Test Pilot data collected and c) the link to the custom policy. 

So that satisfies all the above with just one preamble paragraph and policy link.

Reply:
OK, I think I'm starting to get it. The problem is stating somewhere that the experiment collects the "data collected by all Test Pilot Experiments".

I wonder why it can't be a simple sentence, but at this point it's not an error.

P.S. dev servers seem pretty dead :-\